### ISSUE TITLE
thujohn/pdf-l4

### ISSUE THREAD

**Issue by vishalmote** [thujohn/pdf-l4#43]
I want to add image into pdf report. where I am using one method which return a binary image. this image is displaying properly in html report but not in pdf.
image path are  absolute. 
1)/var/www/images/
2)/var/www/pod_files/
project url /var/www/project/
my method is localhost/project/cntrller/getImage?type=pod&filename=xyz.png

getImage() method returns image xyz.png from pod of type is pod else return image from images folder. [ return readfile($imagedestdir . '/' . $filename);]

but its working fine when I am using html report genration but not working when I am generating pdf report.

Please some one help out

**Comment by thujohn** [comment]
I don't have any problem with images...

You have an example here : https://github.com/thujohn/lab/blob/master/app/views/pdf/image.blade.php

Do you have the same problem with an image in your public folder ?

**Comment by vishalmote** [comment]
yes I am facing the same issue when the images are in public folder
